### DISCUSSION TITLE
JSON as text on API REST

### DISCUSSION THREAD

**Discussion by llermaly** [microsoft/presidio#1091] [Q&A] [answered]
Is JSON supported in HTTP rest? 

I tried sending JSON with no luck (the field names are detected as PII as well, which I want to avoid) , also would like to set keys_to_skip. 

If it is not possible what could be a workaround to expose the functionality via HTTP?

Thanks

**Comment by omri374** [comment, marked as answer]
Hi, Presidio offers a REST API for text, but not for semi-structured/batch data. Duplicate of https://github.com/microsoft/presidio/issues/1045

**Reply by llermaly** [reply]
Thanks! would you then recommend just put Flask on top of the python code to make it available as REST service?
